generate_analysis_graphs.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+from collections import OrderedDict
+
+import altair as alt
```

Comment:
alt was a bit confusing when I was reading this code. Could we consider something a bit more descriptive as the alias? alt_charts?

Reply:
I'm not really a fan of "import as", but did it because the docs did, so assumed it was a standard abbreviation, like import numpy as np or import pandas as pd.

However, if you're also finding it confusing I think I would prefer to change it to "import altair".

Reply:
changed to import altair